configure

```diff
@@ -1466,6 +1470,8 @@ esac
 
 echo "ARCH: ${ARCH}"
 echo "Using arch directory: ${ARCHDIR}"
+echo "Architecture-specific static object files:${ARCH_STATIC_OBJS}"
+echo "Architecture-specific shared object files:${ARCH_SHARED_OBJS}"
```

Comment:
For readability, there should be space after `:`.

Reply:
Those variables already have a space at the beginning based on the way they are generated.

Reply:
At least on modern x86, the lines are really long, over 200 characters. Even with wide-screen aspect ratio, that is very long.